entr.c

```diff
@@ -137,6 +137,13 @@ main(int argc, char *argv[]) {
 	open_max = min(65536, (unsigned)rl.rlim_cur);
 	if (open_max == 0)
 		open_max = 65536;
```

Comment:
If we're testing for `open_max < 65536`, then no need to test for a value of `0`.

Reply:
Yes, except in the case where rlim_cur is initially 0, and setrlimit fails, we'd still want to set it to 65536. I've changed that in the PR.